### PR TITLE
Travel game

### PR DESCRIPTION
**Pull request by srinjoy-26**
worked and completed #366

### CODE REVIEW

**Comment by NOiR-07** [comment]
Update your forked repo

**Comment by srinjoy-26** [comment]
> Update your forked repo

Sir I need to create another new pull request?

**Comment by NOiR-07** [comment]
> > Update your forked repo
> 
> Sir I need to create another new pull request?

you can

**Comment by srinjoy-26** [comment]
> > > Update your forked repo
> > 
> > 
> > Sir I need to create another new pull request?
> 
> you can

Sir if this can be merged please merge it

File: index.html [hunk 1]
```diff
@@ -752,6 +752,17 @@ <h5 class="card-title">Transpose-Matrix Calculator</h5>
         </div>
         <!--End of Transpose-Matrix Calculator website-->
 
+          <!-- This div is for travel game -->
+          <div class="card card1">
+            <img class="card-img-top" src="assets/Images/travel_game/travel.png" alt="Travel game">
+            <div class="card-body">
+                <h5 class="card-title">Travel game</h5>
+                <p>Travel the endless path..</p>
+                <a href="../ZeroOctave-Javascript-Projects/Public/Travel game.html" target="_blank" class="btn btn-outline-primary">Try Now</a>
```
Comment (Astrodevil) [inline index.html:955]: Linkage is wrong, make it correct.

**Pull request merged**